### DISCUSSION TITLE
General questions

### DISCUSSION THREAD

**Discussion by ratsalad** [devmfc/debian-on-amlogic#40] [Q&A]
I was just wondering about a few things that someone might be able to answer if they have anything to add.

If configuration changes are made to the configuration on a flash drive are those changes written to emmc during that process?

Anyone try or get their box running with a hotspot?  The other issue for me is that the wifi script only supports secured wifi and not open wifi.

I have a X96 box with the red led only on and when I try to reflash it always fails during the initial partitioning process so I assume that the emmc or something closely related is damaged hence the red led.

**Comment by devmfc** [comment]
> 
> If configuration changes are made to the configuration on a flash drive are those changes written to emmc during that process?

If you mean during the 'install script' 
, then yes, it will be copied.

> Anyone try or get their box running with a hotspot?  The other issue for me is that the wifi script only supports secured wifi and not open wifi.

'My wifi-connect script' is just an example to quickly test wifi, google for all other use cases ( maybe look inside the script for some google key words)

Hotspot:
It is just Debian, so if the wifi driver supports it you can configure a hotspot. 

 
[]()
> I have a X96 box with the red led only on and when I try to reflash it always fails during the initial partitioning process so I assume that the emmc or something closely related is damaged hence the red led.

Strange. My X96 box has a simple gpio led, which is red or blue depending on the gpio state. This state is set during kernel boot with a setting in the dtb for the X96 box, so it will turn blue during boot. 

Which box-config do you use?

It's very unlikely that your red led has anything todo with any failed hardware. It doesn't work like that.
